src/main/java/ch/njol/skript/effects/EffElytraBoostConsume.java

```diff
@@ -0,0 +1,59 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import com.destroystokyo.paper.event.player.PlayerElytraBoostEvent;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Consume Boosting Firework")
+@Description("Prevent the used firework used in an 'elytra boost' event to be consumed.")
+@Examples({
+	"on elytra boost:",
+		"prevent the boosting firework from being consumed"
+})
+@RequiredPlugins("Paper")
+@Since("INSERT VERSION")
+public class EffElytraBoostConsume extends Effect {
+
+	private static final boolean ELYTRA_BOOST_EXISTS = Skript.classExists("com.destroystokyo.paper.event.player.PlayerElytraBoostEvent");
```

Comment:
Since this is only used in the static field, you can just check this condition there